samples/httpbin/httpbin.api

```diff
@@ -1,10 +1,17 @@
 note 'Httpbin service - simply returns all data you sent back to you.'
 
+type 'customType' spec {
+  description 'A custom type'
+
+  from('json') { asGeneric() }
+  to('json') { asGeneric() }
```

Comment:
This is an example that does integration testing.

@katyakhyl you will need to add these lines to your custom types in order to get it working once the PR is merged.